### PR TITLE
line: add Chinese translation

### PR DESCRIPTION
**Pull request by legendqi**
- [ ] The page (if new), does not already exist in the repository.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by blueskyson** [comment]
@legendqi Thank you for adding new page.

File: pages.zh/linux/line.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# line
+
+> 读取单行输入.
```
Comment (blueskyson) [inline pages.zh/linux/line.md:3]:
```suggestion
> 读取单行输入。
```

File: pages.zh/linux/line.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+- 读取输入:
```
Comment (blueskyson) [inline pages.zh/linux/line.md:5]:
```suggestion
- 读取输入：
```

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @legendqi

**Pull request merged**